### PR TITLE
65 66 search page

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#66 #65

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/modules/skills/model/find.js [hunk 1]
```diff
@@ -8,4 +9,66 @@ const findSkillById = async (id) => {
...
+  `;
+
+  const res = await query(sql, [tool, ...tasksArray]);
+  return res.rows;
+};
+
+const getPopularSkillsAndActivities = async () => {
+  const sql = `
+    SELECT
+      popularity,
```
Comment (ashatat) [inline server/modules/skills/model/find.js:47]: You don't need popularity also here

**Review by ashatat — CHANGES_REQUESTED** [review]
Text overlap
image


Awesome work 👍

File: server/modules/skills/model/find.js [hunk 2]
```diff
@@ -8,4 +9,66 @@ const findSkillById = async (id) => {
...
+
+  const tasksQueryText = tasksArray
+    .map((v, i) => `to_tsquery($${i + 2})`) // keep the first param ($1) for the "tool" and start form $2
+    .join(' || '); // matches any word in the task
+
+  const sql = `
+    SELECT
+      id,
+      title,
+      description,
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply server/modules/skills/model/find.js:23]: done

File: server/modules/skills/model/find.js (hunk 1 above)
Comment (RamyAlshurafa) [reply server/modules/skills/model/find.js:47]: done

**Review by ashatat — APPROVED** [review]

**Pull request merged**